en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,349 @@
+
+   Then you can mount the camera with double sided tape to the wall.
+
+### Flight Controller
+
+The Flight Controller can be installed above the ESC.
+
+#### Pixhawk 6c/6c mini
+
+If a Pixhawk 6c or 6c mini is used, simply stick the flight controller with double sided tape into place.
+
+#### Skynode
+
+If a Skynode is used:
+
+1. Place it at the on top of the ESCs and mark the 2 rear mounting locations on the injection molded plastic part of the ZMO.
+1. Remove the Skynode from the vehicle and drill 2 holes with a 2.8 mm drill bit into the plastic part.
+   ![Mounting holes for the Skynode in the back](../../assets/airframes/vtol/omp_hobby_zmo_fpv/flight-controller-01.jpg)
+1. Put the Skynode back into place and screw it down with 2x M3x10 screws.
+
+Another option is to add some threaded inserts into the holes.
+Since the injection molded part of the ZMO is very thin, they need to be glued in place.
+
+1. Screw the front Skynode mount with 2x M3x10 screws at the Skynode.
+1. Then add some 5 min epoxy at the bottom of the mount and put a weight on top of the Skynode until the glue is cured.
+   To access the 2 mounting screws at the front, poke 2 holes from the top through the foam.
+
+   ![Skynode mount in the front](../../assets/airframes/vtol/omp_hobby_zmo_fpv/flight-controller-02.jpg)
+
+### Antennas and RC Receiver
+
+:::note
+If a Skynode is installed the LTE can be used as telemetry and video link.
+If a Pixhawk is used a different [telemetry link](../telemetry/README.md) will be needed.
+An inexpensive example would be a [SiK Telemetry Radio](../telemetry/sik_radio.md).
+:::
+
+1. One LTE antenna can be installed on the bottom of the vehicle.
+   For that you can feed the antenna wire through the opening for the ESC heat-sink.
+
+   ![LTE antenna 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lte-antenna-01.jpg)
+
+1. The second antenna can be installed on the inside of the vehicle on the left side of the battery compartment.
+   The RC receiver can also be placed at the left side of the battery compartment.
+
+   ![LTE antenna 2 and RC receiver](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lte-antenna-02.jpg)
+
+## Software Setup
+
+### Select Airframe
+
+1) Open QGC and go under vehicle setup.
+1) Select [Generic Quadplane VTOL](https://docs.px4.io/main/en/airframes/airframe_reference.html#vtol_standard_vtol_generic_standard_vtol) under the [Airframe](https://docs.px4.io/main/en/config/airframe.html) tab and click "Apply and Restart"
+1) Select the [Parameters](https://docs.px4.io/main/en/advanced_config/parameters.html#finding-updating-parameters) tab and click on "Tools" in the top right corner. Select "Load from file..". Download the [parameter file](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO.params) and upload it to the flight controller.
+1) reboot the vehicle

```

Comment:
Markdown expects lists to look like this `1.`  (though you can order them, you don't have to)
```suggestion
1. Select the [Parameters](https://docs.px4.io/main/en/advanced_config/parameters.html#finding-updating-parameters) tab and click on "Tools" in the top right corner. Select "Load from file..". Download the [parameter file](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO.params) and upload it to the flight controller.
1. Reboot the vehicle.
```